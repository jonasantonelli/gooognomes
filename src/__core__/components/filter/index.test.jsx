/**
 * @jest-environment jsdom
 */
import React from 'react';
import { shallow } from 'enzyme';
import Filter from './index.jsx';

describe('Loadin Component', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Filter />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should contain title class', () => {
    expect(wrapper.exists('.filter')).toEqual(true);
  });

});
