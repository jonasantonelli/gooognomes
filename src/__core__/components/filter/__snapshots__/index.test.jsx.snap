// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loadin Component should match snapshot 1`] = `
<div
  className="filter"
>
  <Age
    handleChange={[Function]}
    maxAge={100}
    minAge={0}
    range={
      Object {
        "max": 100,
        "min": 0,
      }
    }
  />
  <Professions
    handleCheck={[Function]}
    professions={Array []}
  />
  <button
    button="button"
    className="button button--primary"
    onClick={[Function]}
  >
    Apply filter
  </button>
  <button
    button="button"
    className="button button--default"
    onClick={[Function]}
  >
    Clear filter
  </button>
</div>
`;
